cns/service.go

```diff
@@ -316,3 +327,20 @@ func (service *Service) SendErrorResponse(w http.ResponseWriter, errMsg error) {
 	err := acn.Encode(w, &resp)
 	logger.Errorf("[%s] %+v %s.", service.Name, &resp, err.Error())
 }
+
+// TLSVersionNumber returns the version number for the provided TLS version name
+// (e.g. 0x0301)
+func TLSVersionNumber(versionName string) (uint16, error) {
+	switch versionName {
+	case "TLS 1.0":
+		return tls.VersionTLS10, nil
+	case "TLS 1.1":
+		return tls.VersionTLS11, nil
+	case "TLS 1.2":
+		return tls.VersionTLS12, nil
+	case "TLS 1.3":
+		return tls.VersionTLS13, nil
+	default:
+		return 0, errors.Wrap(errBootConfig, "unsupported TLS version name")
```

Comment:
"configuration error" isn't adding a whole lot of information here. I think you can just have `unsupported TLS version name` as the constantized error, then include what you tried to parse (the `versionName` arg) in the `Wrapf`.